pavelib/quality.py

```diff
@@ -740,36 +740,16 @@ def _lint_output(linter, count, violations_list, is_html=False, limit=0):
 
         return ''.join(lines)
 
```

Comment:
Just to be clear, is the reason for removing the diff report for pep8 because we expect violations to be 0 on master and therefore the diff report (for pep8) would be the same as the the quality report?

Reply:
exactly.